protocol-units/mempool/move-rocks/src/lib.rs

```diff
@@ -469,14 +471,14 @@ pub mod tests {
 		let path = temp_dir.path().to_str().unwrap();
 		let mempool = RocksdbMempool::try_new(path)?;
 
-		let transaction1 = MempoolTransaction::at_time(Transaction::new(vec![1], 0), 2);
+		let transaction1 = MempoolTransaction::at_time(Transaction::new(vec![1], 0, 0), 2);
```

Comment:
`Transaction::new` now has two integer arguments that are easy to mix up. Time to introduce a builder API?

Reply:
I can add one to prevent introducing bugs going forward. Should we remove the `::new()` method though? That might be too time consuming right now.

Reply:
It can be done so that `new` captures the essential data and assigns default priority. The builder will start with the data vector, and get the named setters for priority and maybe for binding the sequence number as well.